src/OrchardCore/OrchardCore.DisplayManagement/DisplayManager.cs

```diff
@@ -78,7 +78,7 @@ ILayoutAccessor layoutAccessor
             return shape;
         }
 
-        public async Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, string group = null)
+        public async Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, bool isNew = false, string group = null)
         {
```

Comment:
Why here the default value is false and below it is true for the `IDisplayManager<TModel>`.